tests/integration/components/flexberry-field-test.js

```diff
@@ -1,26 +1,332 @@
+import Ember from 'ember';
+
+import I18nService from 'ember-i18n/services/i18n';
+import I18nRuLocale from 'ember-flexberry/locales/ru/translations';
+import I18nEnLocale from 'ember-flexberry/locales/en/translations';
+
 import { moduleForComponent, test } from 'ember-qunit';
 import hbs from 'htmlbars-inline-precompile';
 
 moduleForComponent('flexberry-field', 'Integration | Component | flexberry field', {
-  integration: true
+  integration: true,
+
+  beforeEach: function () {
+    this.register('locale:ru/translations', I18nRuLocale);
+    this.register('locale:en/translations', I18nEnLocale);
+    this.register('service:i18n', I18nService);
+
+    this.inject.service('i18n', { as: 'i18n' });
+    Ember.Component.reopen({
+      i18n: Ember.inject.service('i18n')
+    });
+
+    // Set 'ru' as initial locale.
+    this.set('i18n.locale', 'ru');
+  }
+});
+
+test('it renders properly', function(assert) {
+  assert.expect(4);
+
+  // Render component.
+  this.render(hbs`{{flexberry-field
+    class=class
+  }}`);
+
+  // Retrieve component.
+  let $component = this.$().children();
+
+  // Check wrapper <div>.
+  assert.strictEqual($component.prop('tagName'), 'DIV', 'Component\'s wrapper is a <div>');
+  assert.strictEqual($component.hasClass('flexberry-field'), true, 'Component\'s wrapper has \' flexberry-field\' css-class');
+  assert.strictEqual($component.hasClass('ui'), true, 'Component\'s wrapper has \'ui\' css-class');
+  assert.strictEqual($component.hasClass('field'), true, 'Component\'s wrapper has \'field\' css-class');
+});
+
+test('label mode works properly', function(assert) {
+  assert.expect(3);
+
+  // Render component.
+  this.render(hbs`{{flexberry-field
+    class=class
+    label=label
+  }}`);
+
+  // Check that label attribute doesn't exist now.
+  this.set('label', null);
+  assert.strictEqual(this.get('label'), null, 'Component\'s hasn\'t wrapper is a <lable>');
```

Comment:
Typo `<lable>`. Possible comment is "Component's wrapper hasn't inner `<label>` by default".